app/core/math.js

```diff
@@ -25,7 +25,7 @@ export const isNumber = (value: string | number): boolean => {
 
 export const multiplyNumber = (x: string | number, y: string | number) =>
   toBigNumber(x)
-    .mul(y)
+    .mul(toBigNumber(y))
```

Comment:
This was throwing an error when price info was too large of a decimal so I added the invocation of `toBigNumber`